cdk/apps/ica_credentials/secrets/runtime/notify_slack/notify_slack.py

```diff
@@ -9,6 +9,8 @@
 def get_aws_account_name(id: str) -> str:
     if id == "472057503814":
         return "prod"
+    elif id == "455634345446":
+        return "stg"
```

Comment:
are we going to be consistent with this throughout everywhere? (as in is this setting the precedent for all projects and how we shorten staging?)